functions-and-scope.js

```diff
@@ -16,6 +15,37 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+/* stap 1: Array volledig checken met loop.
+   Stap 2: conditie gecheckt worden of het cijfer in de array 8 en hoger is dan 8
+   stap 3; Als het true is, wordt er een waarde ++ toegevoegd zodat ze geteld kunnen worden
+   stap 4 : Het resultaat moet worden gelogd in eht antwoord
+ */
+
+const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
+let cumlaudeGrades = 0;
+
+//functie voor het checken of de grades groter zijn of gelijk + teller
+function checkNumberCumlaude(grades){
+    if (grades === 8) {
+        return cumlaudeGrades++
+    }
+    if (grades === 9) {
+        return cumlaudeGrades++
+    }
+    if (grades === 10) {
+        return cumlaudeGrades++
+    }
```

Comment:
Je zou dit kunnen versimpelen:
```suggestion
    if (grades >= 8) {
        return cumlaudeGrades++
    }
```